spec/support/stub_google_oauth2.rb

```diff
@@ -0,0 +1,17 @@
+# frozen_string_literal: true
+
+module StubGoogleOauth2
+  def stub_env_for_omniauth
+    request.env["devise.mapping"] = Devise.mappings[:user]
```

Comment:
Nitpick: Let's use single quotes and instead of using arrow notation for the hash object, let's use colon notation.